File: atcoder.jp/contests/arc020/tasks/arc020_1/main.go
package main

import (
	"fmt"
	"math"
)

func Compare(x, y float64) string {
	if x < y {
		return "<"
	}
	if x > y {
		return ">"
	}
	return "="
}

func main() {
	var A, B float64
	fmt.Scanf("%g %g", &A, &B)

	var ans string
	switch Compare(math.Abs(A), math.Abs(B)) {
	case "<":
		ans = "Ant"
	case "=":
		ans = "Draw"
	case ">":
		ans = "Bug"
	}

	fmt.Println(ans)
}

File: atcoder.jp/contests/arc020/tasks/arc020_1/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int A, B;
  cin >> A >> B;

  string ans;
  if (abs(A) < abs(B)) {
    ans = "Ant";
  } else if (abs(A) == abs(B)) {
    ans = "Draw";
  } else {
    ans = "Bug";
  }

  cout << ans << endl;
}
